packages/admin-stories/src/docs/form/Overview.stories.mdx

```diff
@@ -1,15 +1,295 @@
-import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import { Meta, Story, Canvas, Source } from "@storybook/addon-docs";
+
+import dedent from "ts-dedent";
 
 <Meta title="Docs/Form/Overview" />
 
 # Form Overview
 
-For information on how to validate your form, see [Form Validation](/docs/docs-form-validation--page).  
-For information on how to implement different layout-types of your form, see [Form Layout](/docs/docs-form-layout--page).
+Forms enable users to submit information to a web application. Therefore, forms belong to the essential features of every admin application.
+
+In Comet Admin, React Final Form (RFF) is used for building forms. You can find more information on this framework in the [React Final Form Docs](https://final-form.org/docs/react-final-form/getting-started).
+
+## When (not) to use Final Form
+
+Final Form is strongly recommended if you
+
+-   are building an extensive or complicated form
+-   want to submit the form data to an API
+
+However, Final Form might not be sensible for every use case.
+Final Form is likely overkill if you want to create a solitary input field (e.g., a single select field for client-side filtering).
+In this case, you can use a plain HTML input combined with a React state.
+
+## `FinalForm` Component
+
+Comet Admin provides a `FinalForm` component.
+It's built on top of React Final Form's [`Form` component](https://final-form.org/docs/react-final-form/api/Form).
+
+There are two required properties:
+
+-   `mode` can be `add` (for creating a new resource) or `edit` (for editing an existing resource)
+-   `onSubmit()` handles the submission of form values to the API
+
+<Canvas>
+    <Story id="stories-form-finalform--basic-finalform" />
+</Canvas>
+
+### Submit Mutation Best Practices
```

Comment:
While this section is a good idea for sure, it doesn't belong into an "Overview"